Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsPage.cs

```diff
@@ -25,20 +25,7 @@
 namespace Microsoft.NodejsTools.Options {
     [ComVisible(true)]
     public class NodejsIntellisenseOptionsPage : NodejsDialogPage {
-        private static readonly Version _typeScriptMinVersionForES6Preview = new Version("1.6");
```

Comment:
We're only targetting people with TS installed, both VS2015 and VS15. Based on our telemetry, this is the vast majority of users (we don't actually have a breakdown of who is using the old editor because they don't have TS vs. they switched back manually, but the entire set of 1.2 users on our old IntelliSense engine is miniscule)

I agree we should provide a better experience so that people know they need TS thought. Right now, we just will prompt them on launch in `GetNodejsLanguagePreferencesFromTypeScript`. This does not check TS version however